collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/BackgroundAudioRecordingTest.java

```diff
@@ -154,6 +154,17 @@ public void fillingOutForm_doesntShowStopOrPauseButtons() {
                 .assertContentDescriptionNotDisplayed(R.string.pause_recording);
     }
 
+    @Test
+    public void pressingBackWhileRecording_andClickingSave_exitsForm() {
+        rule.mainMenu()
+                .enableBackgroundAudioRecording()
+                .copyForm("one-question-background-audio.xml")
+                .startBlankForm("One Question")
+                .closeSoftKeyboard()
+                .pressBack(new SaveOrIgnoreDialog<>("One Question", new MainMenuPage(rule), rule))
+                .clickSaveChanges();
```

Comment:
Don't need to verify anything about the save here as that's handled/tested in `FormSaveViewModel`